### ISSUE TITLE
Looking for install documentation

### ISSUE THREAD

**Issue by Islast** [mozilla-iot/tplink-adapter#25]
Hi, I've installed all the requirements listed in the README and I have gateway installed. What now? Should I clone the repo and run package.sh?

**Comment by benfrancis** [comment]
If you you just want to install it to use it, you can do so through the gateway UI.

Main menu -> Settings -> Add-ons -> +

Is that what you were looking for?

**Comment by Islast** [comment]
Yes, that's what I'm looking for. Do I need to clone the git repo first?

**Comment by Islast** [comment]
I should mention, tplink-adapter is not in the list of available addons when I use the UI

**Comment by mrstegeman** [comment]
What platform are you running on? This add-on should be installable as long as `python3 --version` returns 3.5.x, 3.6.x, or 3.7.x.

Regardless, you're correct. If you want to install this manually, do the following:
* Follow the Requirements section of the [README](https://github.com/mozilla-iot/tplink-adapter/blob/master/README.md#requirements) (you may need to adapt for your platform)
* `git clone` this repo into your `~/.mozilla-iot/addons` directory
* Run the `package.sh` script
* Restart the gateway
* In the UI, navigate to _Settings -> Add-ons_ and enable the TP-Link add-on
